### ISSUE TITLE
Deleting the current menu item causes the app to get stuck in an error loop

### ISSUE THREAD

**Issue by stuaxo** [bluesabre/menulibre#71]
To reproduce:  View a menu item, then delete its desktop file in the commandline.

What happens: An error is displayed, and then it returns when you press a button.

What should happen:   The error should be displayed only once, after that the item should probably disappear from the items in the app.

**Comment by bluesabre** [comment]
Duplicate of https://github.com/bluesabre/menulibre/issues/70
